src/Symfony/Component/BrowserKit/Tests/AbstractBrowserTest.php

```diff
@@ -358,6 +358,7 @@ public function testSubmitFormNotFound()
             $this->fail('->submitForm() throws a \InvalidArgumentException if the form could not be found');
         } catch (\Exception $e) {
             $this->assertInstanceOf(\InvalidArgumentException::class, $e, '->submitForm() throws a \InvalidArgumentException if the form could not be found');
+            $this->assertSame('The form doesn\'t contain any button with "Register" as its content, id, value or name.', $e->getMessage());
```

Comment:
I suggest using `expectException` and `expectExceptionMessage` to write this test instead of those manual assertions on a caught exception.